src/device.js

```diff
@@ -0,0 +1,8 @@
+module.exports = async function device(req, res) {
+  let [gaiaCommit] = await Promise.all([
```

Comment:
No need for `Promise.all` if it's only one thing, but I figure you're probably thinking ahead to when this gets more info